benchmarks/pca/datasets/simulated_blobs.py

```diff
@@ -0,0 +1,30 @@
+from benchopt import BaseDataset, safe_import_context
+from benchopt.datasets import make_correlated_data
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+
+
+class Dataset(BaseDataset):
+    name = "Simulated_correlated_data"
+
+    parameters = {
+        "n_samples, n_features": [(1_000_000, 10_000), (10_000, 10_000)],
```

Comment:
I would rather test:

```
"n_samples, n_features": [(1_000_000, 100), (10_000, 10_000)],
```

If `(10_000, 10_000)` is too slow, we can test `(1_000, 10_000)` instead.

Reply:
oops that was a mistake thanks